week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,9 +7,21 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
 
+
+function convertToUSD(price) {
+  return (price+ foreignTransaction)*1.4;
+}
+function convertToBRL(price) {
+  return (price+foreignTransaction)*4.99:
```

Comment:
did this give the expected answer? I think this should be:
`return (price + foreignTransaction(price)*4.99;`

If you are calling a function you will have parentheses ( and ). 

Also, a small point, try and be consistent with things like spaces. e.g. 1 + 2 , not sometimes 1+ 2 and other times 1 +2 etc. Just keeps things more readable.